### PR TITLE
Makefile: Add redirect support

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,3 +14,24 @@ htmlproofer: clean _site
...
 	# Rerun htmlproofer across _all_ files, but ignore failure, allowing us to notice legacy docs issues without failing CI
 	-docker run -ti -e JEKYLL_UID=`id -u` --rm -v $$PWD/_site:/_site/ quay.io/calico/htmlproofer /_site --assume-extension --check-html --empty-alt-ignore
+
+strip_redirects:
+	find -name '*.md' -exec sed -i'' '/redirect_from:/d' '{}' \;
+
+
+add_redirects_for_latest: strip_redirects
+ifndef VERSION
+	$(error VERSION is undefined - run using make add_redirects_for_latest VERSION=vX.Y.Z)
```
Comment (djosborne) [inline Makefile]: Can you change this to `VERSION=vX.Y`. We don't use the revision here

Comment (tomdee) [inline Makefile]: Done

**Comment by djosborne** [comment]
I'm seeing that visiting `http://0.0.0.0:4000/latest/` after running this is printing the contents of the rendered `_site/latest` directory. We must be missing something here...

**Comment by djosborne** [comment]
@tomdee I'm willing to discuss solutions here. For example, that .html can probably be made a .md, especially now that we can use `redirect_to` in it.

**Comment by tomdee** [comment]
@djosborne I updated the Makefile to cover .html files too

**Pull request merged**